### ISSUE TITLE
Analogy implementation

### ISSUE THREAD

**Issue by arthurcgusmao** [thunlp/OpenKE#45]
Hi,

I have a question about ANALOGY's implementation in PyTorch: in [line 44](https://github.com/thunlp/OpenKE/blob/35bc6a90fd6413af27877e759413b023bb6de57f/models/Analogy.py#L44) we see that the softplus activation function is used. Should not it be the sigmoid function however? In the [original implementation](https://github.com/quark0/ANALOGY/blob/master/main.cpp#L272) we see that a sigmoid was used.

Btw, I opened a [pull request](https://github.com/thunlp/OpenKE/pull/44) on the implementation of Analogy in TensorFlow, following the implementation in PyTorch.

Thanks for your attention and support.

**Comment by THUCSTHanxu13** [comment]
https://www.tensorflow.org/versions/master/api_docs/python/tf/log_sigmoid shows that "Specifically, y = log(1 / (1 + exp(-x))). For numerical stability, we use y = -tf.nn.softplus(-x)."

**Comment by arthurcgusmao** [comment]
Great! Thanks for your help.
